### ISSUE TITLE
training the neural network should be much more general 

### ISSUE THREAD

**Issue by sg-s** [sg-s/crabsort#19]
Given some time series with labelled spikes,

1. remember the spike times and labels
2. keep dropping the spike prominence detection till we double the # of spikes
3. we have now generated our negative training dataset

open problems: 

1. how do we know whether to use + or - prom?

**Comment by sg-s** [comment]
done
